skins/cat17/address-form/src/store.ts

```diff
@@ -0,0 +1,127 @@
+import Vue from 'vue';
+import Vuex, { StoreOptions } from 'vuex';
+import JQueryTransport from '../../src/app/lib/jquery_transport';
+import { Validity } from './lib/validation_states';
+import { Helper } from './mixins/helper';
+import { AddressState, Payload, Form, FormData, inputField,ValidationResult } from './types';
+
+const VALIDATE_INPUT: string = 'VALIDATE_INPUT';
+const MARK_EMPTY_FIELD_INVALID: string = 'MARK_EMPTY_FIELD_INVALID';
+const BEGIN_ADDRESS_VALIDATION: string = 'BEGIN_ADDRESS_VALIDATION';
+const FINISH_ADDRESS_VALIDATION: string = 'FINISH_ADDRESS_VALIDATION';
+const ADDRESS_FIELD_VALIDATOR_NAMES: Array<string> = [
+	'addressType',
+	'salutation', 'title', 'firstName', 'lastName',
+	'companyName',
+	'street', 'postcode', 'city', 'country'
+]
+let transport = new JQueryTransport();
```

Comment:
This will make the store hard to test, because now it depends on a networking component. Please inject a  TransportInterface in the constructor instead and initialize the transport and store in `main.js`